en-EN.js

```diff
@@ -191,74 +191,74 @@ module.exports = {
         },
         graph_types: {
             server_messages: {
-                title: "Messages Envoyés",
-                description: "Graphique résumant l'activité par messages du serveur.",
-                graph_name: "Résumé de l'activité par message"
+                title: "Sent Messages",
+                description: "Graph by server activity in messages.",
```

Comment:
"Graph by" me paraît bizarre comme tournure de phrase, non ?
Utiliser "summarizing" (grosso modo le verbe "summarize") peut être une traduction plus précise *(et du coup même chose pour les entrées du dessous)* 

Et "message activity" est peut-être mieux que "activity in messages"